src/domain/collaboration/callout/callout.service.ts

```diff
@@ -122,6 +124,22 @@ export class CalloutService {
     return await this.calloutRepository.save(callout);
   }
 
+  public async updateCalloutPublishInfo(
+    callout: ICallout,
+    publisherID: string,
+    publishedTimestamp?: number
+  ): Promise<ICallout> {
```

Comment:
1. Use a proper dto for input type.
2. The implementation suggests that PublisherID is always changed and publishDate cannot be changed without changing the publisherID. Do we want it like this or make both optional so that we can choose what we want to change?

Reply:
1. No, this is a method inside the servie, there is a DTO used on the mutation which is where it is required. We do not pass around DTO
2. Ok, both can be optional